### DISCUSSION TITLE
HSB vs HSV

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#726] [Polls]
Just curious!

**Comment by ghost** [comment]
HSL

**Reply by CST1229** [reply]
HSL is a different color space; HSV and HSB are two ways to refer to the same color space.

**Reply by ghost** [reply]
I thought HSB meant HSL.

**Reply by BlueDome77** [reply]
[Nope :(](https://en.wikipedia.org/wiki/HSL_and_HSV)

**Reply by PURPL3-CAT** [reply]
I think HSV is better because the B in HSB can be mistaken for 'blue'  especially if its on its own somewhere.
